src/app/_components/ListboxOption.tsx

```diff
@@ -31,12 +31,12 @@ function OptionContent({ option, counts }: ListboxOptionProps) {
 
 export function ListboxOption({ option, counts }: ListboxOptionProps) {
   return (
-    <Listbox.Option value={option.id} as={Fragment}>
-      {({ active, selected }) => (
+    <HeadlessUIListboxOption value={option.id} as={Fragment}>
+      {({ focus, selected }) => (
```

Comment:
Instead of using render props, how about we use `data-selected` and `data-focus` pseudo classes?

https://headlessui.com/react/listbox#listbox-option